src/components/ForgotPass/ForgotPass.jsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react"
+import { useRef, useState } from "react"
+import { Link } from "react-router-dom"
+import { Form, Button, Card, Alert } from "react-bootstrap" 
+import { useAuth } from "../../contexts/AuthContext"
+import "./ForgotPass.css"
+
+export default function ForgotPass() {
+    const emailRef = useRef()
+    const { resetPassword } = useAuth()
+    const [loading, setLoading] = useState(false)
+    const [message, setMessage] = useState("")
+
+    const handleSubmit = async (e) => {
+        e.preventDefault()
+
+        try {
+            setMessage("")
+            setLoading(true)
+            await resetPassword(emailRef.current.value)
+            setMessage("Check your inbox for further instructions")
+        } catch {
+            //TODO: error handling
+            console.log("Failed to reset password")
+        }
+        setLoading(false)
+    }
+
+    return (
+        <div className="home">
+            <Card>
+            <Card.Body>
+                <h2>Reset Password</h2>
+                {message && <Alert variant="success">{message}</Alert>}
```

Comment:
Do you set message to null? If not, the alert will always be there right?

Reply:
When Alert is an empty string, nothing shows up. I think bootstrap does that check for me